docs/src/main/asciidoc/logging.adoc

```diff
@@ -249,6 +247,33 @@ test {
 	systemProperty "java.util.logging.manager", "org.jboss.logmanager.LogManager"
 }
 ----
+== Running `@QuarkusTest` from an IDE
+
+Most IDEs offer the possibility to run a selected class as JUnit test directly, in which case the above workaround will not be applied. You will need to apply it in the settings of your chosen IDE.
+
+=== Eclipse separate JRE definition
+
+Copy your current "Installed JRE" definition into a new one, where you will add the workaround as a new VM argument 
+`-Djava.util.logging.manager=org.jboss.logmanager.LogManager`. Use this JRE definition as your Quarkus project targeted runtime and the workaround will be applied to any "Run as JUnit" configuration.
+
+=== VSCode "run with" configuration
+
+The `settings.json` placed in the root of your project directory or in the workspace will need the workaround in your test configuration:
+[source, json]
+----
+"java.test.config": [
+    {
+        "name": "quarkusConfiguration",
+        "vmargs": [ "-Djava.util.logging.manager=org.jboss.logmanager.LogManager ..." ],
+        ...
+    },
+  ...
+]
+----
+
+=== IntelliJ JUnit template
+
+In the same screen where you edit your run configuration you can edit the JUnit template to add the workaround Java VM argument `-Djava.util.logging.manager=org.jboss.logmanager.LogManager`.
```

Comment:
I am not an IntelliJ user but the Maven integration should automatically pick up `systemPropertyVariables` from the surefire config in pom.xml.

Reply:
I took that from the issue discussion, as I'm not an IntelliJ user either. Just Eclipse and VSCode are tested.

Reply:
I can confirm surefire systemPropertyVariables get picked-up by https://github.com/JetBrains/intellij-community/blob/master/plugins/maven/src/main/java/org/jetbrains/idea/maven/execution/MavenJUnitPatcher.java#L162 , so it should work out of the box.